src/main/kotlin/io/libp2p/pubsub/gossip/GossipRouter.kt

```diff
@@ -59,6 +55,9 @@ open class GossipRouter @JvmOverloads constructor(
     subscriptionTopicSubscriptionFilter: TopicSubscriptionFilter = TopicSubscriptionFilter.AllowAllTopicSubscriptionFilter()
 ) : AbstractRouter(subscriptionTopicSubscriptionFilter, params.maxGossipMessageSize) {
 
+    val acceptRequestsWhitelistThresholdScore = 0
+    val acceptRequestsWhitelistMaxMessages = 128
+    val acceptRequestsWhitelistDuration = 1.seconds
```

Comment:
Any particular reason to choose these specific params?  Might be worth a comment.

Reply:
The idea was that those param values
- from one side are pretty small and safe: peer unlikely be able to drop its score to `graylist` with 128 messages. But even if so then it's not critical to accept some extra messages before blocking - not too much space for DoS here
- from the other side param values are pretty high to yield good performance gain

Will add these comments to the code